src/xref2/compile.ml

```diff
@@ -16,9 +16,9 @@ let type_path : Env.t -> Paths.Path.Type.t -> Paths.Path.Type.t =
   match p with
   | `Resolved _ -> p
   | _ -> (
-      let cp = Component.Of_Lang.(type_path empty p) in
+      let cp = Component.Of_Lang.(type_path (empty ()) p) in
       match Tools.resolve_type_path env cp with
-      | Ok p' -> `Resolved (Cpath.resolved_type_path_of_cpath p')
+      | Ok p' -> `Resolved Lang_of.(Path.resolved_type (empty ()) p')
```

Comment:
Why this change ?

Reply:
We had two ways to convert from `Cpath.*` to `Path.*` - I deleted the ones in `Cpath` and added the memoisation to the other (in `Lang_of`).